manager/logbroker/broker.go

```diff
@@ -42,7 +42,7 @@ type LogBroker struct {
 	subscriptionQueue *watch.Queue
 
 	registeredSubscriptions map[string]*subscription
-	connectedNodes          map[string]struct{}
+	connectedNodes          map[string]map[*subscription]struct{}
```

Comment:
Let's rename this to `subscriptionsByNode`. The first level of the map is the set of connected nodes, but the data that's ultimately indexed is the set of subscriptions for each node.